target/linux/ath79/image/generic.mk

```diff
@@ -607,6 +619,25 @@ define Device/engenius_ecb1750
 endef
 TARGET_DEVICES += engenius_ecb1750
 
+define Device/engenius_ens202ext
+  SOC := ar9341
+  DEVICE_VENDOR := EnGenius
+  DEVICE_MODEL := ENS202EXT
+  DEVICE_PACKAGES := rssileds kmod-leds-gpio
+  IMAGE_SIZE := 12032k
+  KERNEL := kernel-bin | append-dtb | lzma | uImage lzma -M 0x4f4b4c49
+  LOADER_TYPE := bin
+  LOADER_FLASH_OFFS := 0x230000
+  COMPILE := loader-$(1).bin loader-$(1).uImage
+  COMPILE/loader-$(1).bin := loader-okli-compile
+  COMPILE/loader-$(1).uImage := append-loader-okli $(1) | \
+	pad-to $$$$(BLOCKSIZE) | lzma | uImage lzma
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-squashfs-fakeroot-be | pad-to $$$$(BLOCKSIZE) | \
+	$$(IMAGE/sysupgrade.bin) | engenius_ens202ext-factory $(1) | gzip
```

Comment:
The $(1) is quite uncommon for device nodes, maybe check how other recipes handle this (I think there is something like DEVICE_NAME, which can be used directly in the recipe ...)

Reply:
I took that from #2299 
https://github.com/openwrt/openwrt/pull/2299/files#diff-c231eaae15ac6c91e5a34e4ea305bffaR791

I did have $$(DEVICE_MODEL) at one point, but I thought this looked cleaner, as it has to be written 6 times.

It could be anything though, as long as they are the same. Instead of a variable is it better to just have a string like "ens202ext"?